pyproject.toml

```diff
@@ -38,6 +38,7 @@ dependencies = [
   "sortedcontainers",
   "protobuf>=3.20, <5.0",
   "ddsketch",
+  "httpx",
```

Comment:
this is not a dependency we need - If desired for testing we can put it under dev but we are already bloated in deps as is. Let's nix.

Reply:
Sounds good, how do you feel about excluding our `scripts` directory from mypy checks altogether? IIRC we were already implicitly dependent on httpx and I just needed to make it explicit to make the type checker happy